Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessFieldedMetadata.java

```diff
@@ -705,6 +713,42 @@ public void preserve() throws InvalidMetadataValueException, NoSuchMetadataField
         }
     }
 
+    private void updateDivisionFromProcessDetail(String key, ProcessSimpleMetadata processDetail) throws InvalidMetadataValueException {
+        String simpleValue = extractSimpleValue(processDetail);
+        if (!processDetail.getSettings().isValid(simpleValue, getListForLeadingMetadataFields())) {
+            throw new InvalidMetadataValueException(key, simpleValue);
+        };
+        if (simpleValue == null) {
+            return;
+        }
+        updateDivision(key, simpleValue);
+    }
+
+    private String extractSimpleValue(ProcessDetail value) {
+        if (value instanceof ProcessTextMetadata) {
+            return ((ProcessTextMetadata) value).getValue();
+        } else if (value instanceof ProcessSelectMetadata) {
+            return String.join(" ", ((ProcessSelectMetadata) value).getSelectedItems());
+        }
```

Comment:
Missing to handle `ProcessBooleanMetadata`

Reply:
I added boolean metadata.